### ISSUE TITLE
The 'eject' script results in an error.

### ISSUE THREAD

**Issue by DanielRosenwasser** [wmonk/create-react-app#1]
When I tried to run `yarn run eject`, I'm got the following:

```
error An unexpected error occurred: "ENOENT: no such file or directory, open 'C:\\Users\\drosen\\my-app\\node_modules\\@types\\jest\\index.d.ts'".
```

**Comment by wmonk** [comment]
Thanks for the report, investigating now.

**Comment by wmonk** [comment]
huh, don't seem to be able to reproduce this, do you have a `yarn-error.log`?

**Comment by DanielRosenwasser** [comment]
Here's what I got from trying it again, getting a similar error for `@types/node`:

https://gist.github.com/DanielRosenwasser/a0a6be895c8549bad5f7109524bffe4c

**Comment by wmonk** [comment]
Hopefully this is fixed in [Yarn 18.x.x](https://github.com/yarnpkg/yarn/issues/825#issuecomment-265117081)

**Comment by DanielRosenwasser** [comment]
Jeez, sorry about that - that did it! Though I still get the following at the very end.

```
The batch file cannot be found.
error Command failed with exit code 1.
```

There is no `yarn-error.log`, so it's probably fine. If you want to close this out, go for it.

**Comment by wmonk** [comment]
@DanielRosenwasser no worries at all! Yarn is moving so quickly little things seem to be being patched all the time. I don't have much experience with yarn, and especially yarn on Windows, so don't think I can be much help for the issue above. I did just push a fix for `yarn run test` afer ejecting in 1.0.1, so make sure you update for that!
